test/simulator/timestamp/timestamp_simulator.cpp

```diff
@@ -0,0 +1,13 @@
+#include <iostream>
```

Comment:
I believe we need to add the copyright header on cpp files.

Reply:
Updated in [13e01a3](https://github.com/wiredtiger/wiredtiger/pull/8054/commits/13e01a31a3fa34556a2c37369139bf723852fc54)